src/tetris/ui/scoreboardWidget.ts

```diff
@@ -108,58 +79,107 @@ export default class ScoreboardWidget {
 	//endregion
 
     //region private methods
-    private _clearScoreboard(): void {
-        for (let index = 0; index < this.ranks.length; index++) {
-            this.ranks[index].destroy();
-            this.names[index].destroy();
-            this.scores[index].destroy();
+    private _createTextObjects(): void {
+        for (let index = 0; index < DISPLAYED_PLAYERS; index++) {
```

Comment:
There are several functions looping somehow over `DISPLAYED_PLAYERS`.
I would suggest to introduce a helper function that loops over it and executes custom payload passed in via arguments

Reply:
Could you give an example of how this could work?

Reply:
```
private _applyToAllDisplayedPlayers(payLoad: (index: number) => void) {
    for (let index = 0; index < DISPLAYED_PLAYERS; index++) {
        payload(index);
    }
}

private _someFunction() {
    this._applyToAllDisplayedPlayers(index => {
        // some stuff to do
    });
}